src/EncoderShim.cpp

```diff
@@ -4,6 +4,9 @@
 //
 
 #include "EncoderShim.h"
+#include "Sounder.h"
```

Comment:
@bonzer35 You can refactor out having your beeper triggered within `EncoderShim` completely, and save a lot of heartache.   Move the `beep_activate` call into the click callback in main,   This eliminates completely the necessity to have the extern and whatnot, and keeps this encoder shim more focused.